src/app/pages/products-list/card/card.component.html

```diff
@@ -1,23 +1,19 @@
 <mat-card class="card">
     <mat-card-header>
-        <mat-card-title>9.6" Планшет Dexp Ursus S290 32 ГБ 3G черный</mat-card-title>
-        <mat-card-subtitle>86 $</mat-card-subtitle>
+        <mat-card-title>{{ product?.name }}</mat-card-title>
+        <mat-card-subtitle>{{ product?.price + ' $' }}</mat-card-subtitle>
```

Comment:
```suggestion
        <mat-card-subtitle>{{ product?.price }} $</mat-card-subtitle>
```
Нам предоставляют удобный синтаксис шаблона, грех его не использовать) Всю статичную часть лучше выносить напрямую в шаблон